OutOfSchool/OutOfSchool.DataAccess/Models/Configurations/CompetitiveEventCoverageConfiguration.cs

```diff
@@ -0,0 +1,17 @@
+﻿using Microsoft.EntityFrameworkCore;
+using Microsoft.EntityFrameworkCore.Metadata.Builders;
+using OutOfSchool.Services.Models.CompetitiveEvents;
+
+namespace OutOfSchool.Services.Models.Configurations;
+
+internal class CompetitiveEventCoverageConfiguration : IEntityTypeConfiguration<CompetitiveEventCoverage>
+{
+    public void Configure(EntityTypeBuilder<CompetitiveEventCoverage> builder)
+    {
+        builder.HasKey(x => x.Id);
+
+        builder.HasIndex(x => x.IsDeleted);
+
+        builder.Property(x => x.IsDeleted).HasDefaultValue(false);
```

Comment:
(Code reuse) Same code for all `IKeyed` and `ISoftDeleted` entities. Why not introduce some helper method?

Reply:
Done